src/app/zap-templates/templates/chip/CHIPClustersObjc-src.zapt

```diff
@@ -358,7 +358,11 @@ private:
         return NO;
     }
 
+<<<<<<< HEAD
+    CHIP_ERROR err = self.cppCluster.{{asCamelCased name false}}(callback{{#chip_server_cluster_command_arguments}}, {{asCamelCased label}}{{#if (isByteString type)}}, {{asCamelCased label}}Len{{/if}}{{/chip_server_cluster_command_arguments}});
+=======
```

Comment:
I guess that's the root of what looks like a merge issue in all the `CHIPClustersObjc.mm` files.